gemfiles/rails_4.1.gemfile

```diff
@@ -4,4 +4,4 @@ source "https://rubygems.org"
 
 gem "rails", "~> 4.1.0"
 
-gemspec :path => "../"
+gemspec path: "../"
```

Comment:
these files are generated by appraisal, so did you get a change here or did you make this change?

Reply:
@gsmetal could you tell me about this one?

Reply:
Oh.. I wrote about this and leave it as `Pending` review change 😕

I executed `appraisal install` with newer version of appraisal so it made this change. I tried to add rails 5.2 Appraisal but it had problems with `"table_name is set to 'legacy'"` test (with parentheses in SQL) so I didn't dig into that and removed it.

Reply:
BTW, there are test problems on travis with older versions of rails and ruby caused by bundle 2.0 release.

Reply:
I don't suppose you have a link about mitigating those failures? I assume I can just tell travis to stick to old version or something, or there's a script, but I'm porbably being naively hopeful.

Reply:
No, I don't know quick way to fix all of this new dependencies between new RVM, RubyGems 3.0, Bundler 2.0 and older versions of Ruby and Rails in Travis.

Reply:
I assume you should pin rubygems and bundler version below 3.0 and 2.0 respectively here https://github.com/expectedbehavior/acts_as_archival/blob/87fe14e610bd42f7caa679613b59b01eb8efe620/.travis.yml#L20-L21

Reply:
I dug in on this one on #39 and fixed it there. I'll check on these changes again and merge them if I still agree with them :)